client/.eslintrc

```diff
@@ -40,7 +40,7 @@ rules:
   react/no-did-update-set-state: 0
   react/no-multi-comp: 2
   react/no-unknown-property: 2
-  react/prop-types: 1
+  react/prop-types: 0
```

Comment:
@justin808  did you read the link in my above comment about the github issue from the `eslint-plugin-react`. Someone familiar with the react code should look at this and see if anything new in the code has triggered this issue? Or is still a bug in the plugin. Maybe you folks can try another react-props eslint plugin?